mk/lint.mk

```
@@ -0,0 +1,18 @@
+ADD_PYTHON_ENV := source .venv/bin/activate &&
+GOLANGCI_LINT_VERSION := v1.47.2
+
+.venv:
+	python -m venv .venv && $(ADD_PYTHON_ENV)  pip install -U pip && pip install pre-commit
+
+.PHONY: install-pre-commit
+install-pre-commit: install-golangci-lint .venv ## Install pre-commit linter
+	pip3 install pre-commit
+	pre-commmit install --install-hooks --allow-missing-config
```

```suggestion
	$(ADD_PYTHON_ENV) pip3 install pre-commit
	$(ADD_PYTHON_ENV) pre-commit install --install-hooks --allow-missing-config
```
sorry I forgot to prefix with this to include the local .venv